.sauce/xcuitest.yml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1alpha
+kind: xcuitest
+sauce:
+  region: us-west-1
+  concurrency: 1
+  metadata:
+    name: Testing XCUITEST Support
+    tags:
+      - e2e
+      - release team
+      - other tag
+  build: Release $CI_COMMIT_SHORT_SHA
+
+xcuitest:
+  app: ./tests/apps/xcuitest/SauceLabs.Mobile.Sample.XCUITest.App.ipa
+  testApp: ./tests/apps/xcuitest/SwagLabsMobileAppUITests-Runner.ipa
+
+suites:
+  - name: "saucy xcuitest"
+    testOptions:
+      class: ["SwagLabsMobileAppUITests.LoginTests/testSuccessfulLogin"]
+    devices:
+      # If set, only device by ID will be queried.
+      #- id: <rdc_device_id>
+      - name: "iPhone 11"
+        platformName: iOS
```

Comment:
Can I ever run xcuitests on anything but iOS? If not, why is platformName a configurable option 🤔

Reply:
We can run only for the `iOS`, but I use the same structure as for the `Espresso`. I set `iOS` value even if it's empty. Does it worth adding additional structure only for the `XCUITest` to make it not configurable?

Reply:
It's fine to re-use structs or other data transfer objects. The _user_ doesn't care how you technically handle the execution underneath. But that also means that you shouldn't expose those internals to the user either. Meaning, that if the user doesn't need to set the platformName here (in fact, they can't, since you always replace the value anyway), remove it from the config (examples) as well :) Otherwise you are illustrating and suggesting that specifying the platformName here has an influence on the test itself.

Reply:
I removed `platfromName` from a config and keep in mind that we don't need to include `platfromName` as a required property in our example repo and docs.